A/AMGX/build_tarballs.jl

```diff
@@ -55,9 +62,21 @@ products = [
     LibraryProduct("libamgxsh", :libamgxsh),
 ]
 
-dependencies = [
-    BuildDependency(PackageSpec(name = "CUDA_full_jll", version = "10.0")),
-    Dependency(PackageSpec(name = "CUDA_jll", version = "10.0")),
-]
+dependencies = [RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll"))]
+
+for cuda_version in [v"11.0"], platform in platforms
+    augmented_platform = Platform(arch(platform), os(platform); cuda=CUDA.platform(cuda_version))
+    should_build_platform(triplet(augmented_platform)) || continue
+
+    cuda_deps = [
+        BuildDependency(PackageSpec(name="CUDA_full_jll",
+                                    version=cuda_full_versions[cuda_version])),
+        RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll",
+                                      version=v"0.2"), compat="0.2"),  # avoid pulling in CUDA 12 for now.
```

Comment:
No need for a version bound. AMGX.jl should check if AMGX_jll.is_available(), and if not, notify that e.g. this platform or CUDA version is unsupported (which can be remedied by setting a preference, by calling CUDA.set_runtime_version!)

Reply:
I took this from Libxc btw